src/js/components/Button/doc.js

```diff
@@ -177,6 +181,46 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '24px',
   },
+
+  'button.active.background.color': {
+    description: `Background color when the button is active.`,
+    type: 'string | { dark: string, light: string }',
+    defaultValue: undefined,
+  },
+  'button.active.border.color': {
+    description: 'The border color when the button is active.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: undefined,
+  },
+  'button.active.color': {
+    description: `Label color when the button is active.`,
+    type: 'string | { dark: string, light: string }',
+    defaultValue: undefined,
+  },
+  'button.active.extend': {
+    description: 'Any additional style for a active Button.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },
+  'button.active.default': {
+    description:
+      'Adjustments to the default Button style when the Button is active.',
+    type: '{}',
```

Comment:
In most cases of documentations, we use `object` vs `{}`, something like:
```suggestion
    type: 'object',
```